layouts/guides/list.html

```diff
@@ -7,24 +7,7 @@ <h1>GSA Tech Guides</h1>
     {{ .Content }}
 
     <div class="grid-row grid-gap">
```

Comment:
Hey @brentryanjohnson — I think we can delete the grid divs altogether since we no longer have columns to worry about. So deleting this div and the one below, which will promote the one `  <div class="guides-category" data-category="{{ .Key | urlize }}">` to the top level.